CTP/README.md

```diff
@@ -192,6 +192,42 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
 	* ``test_${Node_Name}.log`` will show the logs of testing based on this server node.
 
 
+- **SQL_By_CCI**
+  - Prepare
+        * ``SQL_By_CCI`` executes ``SQL`` cases by using CCI driver, so checkout ``SQL`` testcase from our GitHub projects or make your own test case
+        * Install CUBRID build and make sure your environment variable of CUBRID is correctly set, the configuration file is same as ``SQL``, please refer to [CTP/conf/sql_by_cci.conf](conf/sql_by_cci.conf) for the details
+        * Environment variables set on test machine:
+
+          ```
+             JAVA_HOME=$java_nstallation_directory
+             CTP_HOME=$HOME/CTP
+          ```
+
+ - Run Tests
+        * For **SQL_By_CCI** test:
+
+          ```
+          $ bin/ctp.sh sql_by_cci -c ./conf/sql_by_cci.conf

```

Comment:
@supler
OK. Then, It'd better to use the 'sql.conf' instead of 'sql_by_cci.conf' , and change sentence in above 'Prepare' section to refer the SQL's Prepare section.